src/python/grpcio/tests/protoc_plugin/beta_python_plugin_test.py

```diff
@@ -244,35 +244,63 @@ class PythonPluginTest(unittest.TestCase):
   methods and does not exist for response-streaming methods.
   """
 
-  def setUp(self):
```

Comment:
I'm going to make the wild guess that it's to attempt to enforce `_gen_protos` being invoked exactly once. If that's the case, I'd prefer using the documented methods mentioned [here](https://docs.python.org/2.7/library/unittest.html#setupclass-and-teardownclass) and [here](https://docs.python.org/2.7/library/unittest.html#setupmodule-and-teardownmodule).